autotest/utilities/test_gdal_viewshed.py

```diff
@@ -213,6 +213,31 @@ def test_gdal_viewshed_all_options(gdal_viewshed_path, tmp_path, viewshed_input)
 ###############################################################################
 
 
+def test_gdal_viewshed_cumulative(gdal_viewshed_path, tmp_path, viewshed_input):
+
+    import numpy as np
```

Comment:
Not all builds have numpy enabled, so:

```suggestion
    np = pytest.importorskip("numpy")
```

Reply:
I was just experimenting at this point. Is there a better way to read the raster data in Python?

Reply:
> Is there a better way to read the raster data in Python?

no, that's the appropriate one

Reply:
Thanks.